### ISSUE TITLE
ImportError: No module named 'requests' in configure.sh

### ISSUE THREAD

**Issue by MadderHat** [LetheanMovement/lethean-vpn#9]
Compiling on AWS t2.micro (CentOS?):
`.............................................++*++*
Traceback (most recent call last):
  File "server/dispatcher/itnsdispatcher.py", line 20, in <module>
    import services
  File "/usr/local/src/lethean-vpn/server/dispatcher/services.py", line 7, in <module>
    from service_mgmt import ServiceMgmt
  File "/usr/local/src/lethean-vpn/server/dispatcher/service_mgmt.py", line 6, in <module>
    import authids
  File "/usr/local/src/lethean-vpn/server/dispatcher/authids.py", line 10, in <module>
    import requests
ImportError: No module named 'requests'`

**Comment by valiant1x** [comment]
Python is missing dependencies. Make sure you install requirements, as mentioned in readme:
`pip3 install -r requirements.txt`

If that still does not solve your issue, then the python bin location is wrong in `env.mk`. Rectify using `./configure.sh`

**Comment by MadderHat** [comment]
I did make sure to install requirements. env.mk shows the correct bin location for python. ./configure.sh --easy --with-wallet-address <wallet-address> still returns No module named 'requests'
pip3:
`lethean-vpn]$ sudo /usr/local/bin/pip3 install -r requirements.txt 
Requirement already satisfied: syslogmp in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 1)) (0.2.2)
Requirement already satisfied: ed25519 in /usr/local/lib64/python3.4/site-packages (from -r requirements.txt (line 2)) (1.4)
Requirement already satisfied: pprint in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 3)) (0.1)
Requirement already satisfied: psutil in /usr/local/lib64/python3.4/site-packages (from -r requirements.txt (line 4)) (5.4.7)
Requirement already satisfied: jsonpickle in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 5)) (0.9.6)
Requirement already satisfied: configargparse in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 6)) (0.13.0)`
env.mk:
`ITNS_PREFIX=/opt/itns/
OPENVPN_BIN=/usr/sbin/openvpn
PYTHON_BIN=/usr/bin/python3
PIP_BIN=/usr/local/bin/pip3
SUDO_BIN=/usr/bin/sudo
HAPROXY_BIN=/usr/sbin/haproxy
OPENSSL_BIN=/usr/bin/openssl`
python:
`$ sudo find -name python3
./var/lib/alternatives/python3
./etc/alternatives/python3
./usr/bin/python3
`

**Comment by valiant1x** [comment]
How exactly are you getting the error in the first message? Please provide the full log as I'm not sure what the antecedent command/action was.

Did you use the [easy-deploy-aws.sh](https://github.com/LetheanMovement/lethean-vpn/blob/master/server/easy-deploy-aws.sh) script?

**Comment by MadderHat** [comment]
I was following the instructions in README.md.
I don't know which log you are asking for and I don't see any in folder.
I removed the directory and started from the beginning. Here's everything that happened:

`[user@aws src]$ sudo git clone https://github.com/LetheanMovement/lethean-vpn.git
Cloning into 'lethean-vpn'...
remote: Counting objects: 944, done.
remote: Compressing objects: 100% (95/95), done.
remote: Total 944 (delta 114), reused 132 (delta 83), pack-reused 765
Receiving objects: 100% (944/944), 225.20 KiB | 9.38 MiB/s, done.
Resolving deltas: 100% (662/662), done.
[user@aws src]$ cd lethean-vpn/
[user@aws lethean-vpn]$ sudo /usr/local/bin/pip3 install -r requirements.txt 
Requirement already satisfied: syslogmp in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 1)) (0.2.2)
Requirement already satisfied: ed25519 in /usr/local/lib64/python3.4/site-packages (from -r requirements.txt (line 2)) (1.4)
Requirement already satisfied: pprint in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 3)) (0.1)
Requirement already satisfied: psutil in /usr/local/lib64/python3.4/site-packages (from -r requirements.txt (line 4)) (5.4.7)
Requirement already satisfied: jsonpickle in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 5)) (0.9.6)
Requirement already satisfied: configargparse in /usr/local/lib/python3.4/site-packages (from -r requirements.txt (line 6)) (0.13.0)
[user@aws lethean-vpn]$ sudo ./configure.sh --easy --with-wallet-address iz5TKdqDeHD3EX5qDPHXkeDPRPFCvUHhraz19QeTKG7E5tMiGkHVYCdeHM3rUf3N6qe2WuugDQ9456hPmSLriWqm2j3nbiM8h
Found OPENVPN_BIN at /usr/sbin/openvpn
Found OPENSSL_BIN at /usr/bin/openssl
Found HAPROXY_BIN at /usr/sbin/haproxy
Found PYTHON_BIN at /usr/bin/python3
Found PIP_BIN at /usr/local/bin/pip3
Found SUDO_BIN at /usr/bin/sudo
Generating with default password!
Generating CA ITNSEasyDeploy
Generating RSA private key, 4096 bit long modulus
.................++
..............++
e is 65537 (0x10001)
Generating crt (name=openvpn,cn=ITNSEasyDeploy.openvpn)
Generating RSA private key, 4096 bit long modulus
....++
..................................................................................................................................................................................................................................++
e is 65537 (0x10001)
Using configuration from /usr/local/src/lethean-vpn/conf/ca.cfg
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 0 (0x0)
        Validity
            Not Before: Sep  3 19:05:31 2018 GMT
            Not After : Sep 13 19:05:31 2019 GMT
        Subject:
            commonName                = ITNSEasyDeploy.openvpn
        X509v3 extensions:
            X509v3 Basic Constraints: 
                CA:FALSE
            Netscape Cert Type: 
                SSL Server
            Netscape Comment: 
                OpenSSL Generated Server Certificate
            X509v3 Subject Key Identifier: 
                86:68:34:0F:E6:4A:5A:51:62:0F:12:33:84:B7:54:09:FA:B5:AE:27
            X509v3 Authority Key Identifier: 
                keyid:BB:D6:52:F4:BA:9D:5C:D5:2C:CA:46:4B:A4:53:59:DD:BF:D9:DB:12
                DirName:/CN=ITNSEasyDeploy
                serial:F3:DB:95:8E:17:4C:50:5F

            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication
Certificate is to be certified until Sep 13 19:05:31 2019 GMT (375 days)

Write out database with 1 new entries
Data Base Updated
writing RSA key
writing RSA key
Generating crt (name=ha,cn=ITNSEasyDeploy.ha)
Generating RSA private key, 4096 bit long modulus
......................++
.....................................................................................................++
e is 65537 (0x10001)
Using configuration from /usr/local/src/lethean-vpn/conf/ca.cfg
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 1 (0x1)
        Validity
            Not Before: Sep  3 19:05:32 2018 GMT
            Not After : Sep 13 19:05:32 2019 GMT
        Subject:
            commonName                = ITNSEasyDeploy.ha
        X509v3 extensions:
            X509v3 Basic Constraints: 
                CA:FALSE
            Netscape Cert Type: 
                SSL Server
            Netscape Comment: 
                OpenSSL Generated Server Certificate
            X509v3 Subject Key Identifier: 
                35:F2:66:59:1A:47:FD:80:95:35:E1:90:EB:16:D9:50:FA:42:08:1E
            X509v3 Authority Key Identifier: 
                keyid:BB:D6:52:F4:BA:9D:5C:D5:2C:CA:46:4B:A4:53:59:DD:BF:D9:DB:12
                DirName:/CN=ITNSEasyDeploy
                serial:F3:DB:95:8E:17:4C:50:5F

            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication
Certificate is to be certified until Sep 13 19:05:32 2019 GMT (375 days)

Write out database with 1 new entries
Data Base Updated
writing RSA key
writing RSA key
Generating DH parameters, 2048 bit long safe prime, generator 2
This is going to take a long time
.....................................+....................................................................................................................................................................................+.....................+.......................................................+..............+....................................................................................................+....................................................................+.........................+....+...........................................+.................................+.......................................................+......................+...............................+.......................................................+.........................................................................................................................................................................................................................................................................................+........................................................................................................................................................+.....................................................................................+..............................................................+....................................................................................................+...................................................................................................................................................................................................................................................................................................................................................+......+.................................................+.....................................................+...................................................................................................................................................................................................................................+..........................................................................................................................................................................................+..................................................................................................................+................................................................................+............................................+.....................+............................................................................................................................................................................................................................+............................................................................................................................................................................................................................................................................................................................+..............................................................+.........................................................+...........................................................................................................+.......................................................+...................................................................................................................+...........................................................................................................+................................................+.............................................................................................................................+...............................................................................................................................+.....................................................................................+..........................................................................................................................................................+..........................................................................+................................................................+............................................................................................................................................................................+..................+..........................................................+...............................+....................................................+..............................+................................................................................................................+..................................................................................................................................................................................................................................................+.......................................................................................................................+..........................................................................................................................................+....................................................................................................................................................................................................................................................................................................................................................+................................................................+.....................................+...............................................................................................................+........................................................................................................................................................................+.....................................................................................................................................................................................................................................+.........................................................................+...........................................................................................................................................................................................................................................+...................................+....................................................................................................+..................................+............................................................................................+............................+.....................+...........+............................+.............................................................................+...........................................................................+.......+........................................................................................................................................................................................................................+..................................................................................................+...........................+....................................................................................................................+.................................+...............................................................................................................................................................................................................................................................................................................................................................................++*++*
Traceback (most recent call last):
  File "server/dispatcher/itnsdispatcher.py", line 20, in <module>
    import services
  File "/usr/local/src/lethean-vpn/server/dispatcher/services.py", line 7, in <module>
    from service_mgmt import ServiceMgmt
  File "/usr/local/src/lethean-vpn/server/dispatcher/service_mgmt.py", line 6, in <module>
    import authids
  File "/usr/local/src/lethean-vpn/server/dispatcher/authids.py", line 10, in <module>
    import requests
ImportError: No module named 'requests'
[user@aws lethean-vpn]$`
